client-js/app/elements/labrad-plot.html

```diff
@@ -70,8 +71,8 @@
       </div>
       <div id="selectButtons">
       <!-- TODO implement these  -->
-        <paper-button disabled>ALL</paper-button>
-        <paper-button disabled>NONE</paper-button>
+        <paper-button on-click="selectAll">ALL</paper-button>
```

Comment:
What do these buttons do? Do they plot all traces or just select all traces in the menu? If the latter, then they should be "Select ALL" and "Select NONE".